examples/resources/snowflake_function_scala/resource.tf

```diff
@@ -31,13 +31,13 @@ resource "snowflake_function_scala" "complete" {
   }
   comment                      = "some comment"
   external_access_integrations = ["external_access_integration_name", "external_access_integration_name_2"]
-  function_definition          = <<EOF
+  function_definition          = <<EOT
```

Comment:
Note: We should address this later, but we use both EOF and EOT in different files.

Reply:
I am pretty sure that now we only have EOT (do you see any files with EOF?)

Reply:
Masking policy example and acc test.